internal/pkg/node/datastructure.go

```diff
@@ -75,7 +75,7 @@ type NetDevs struct {
 	Prefix  string            `yaml:"prefix,omitempty"`
 	Netmask string            `yaml:"netmask,omitempty"`
 	Gateway string            `yaml:"gateway,omitempty"`
-	Default string            `yaml:"default,omitempty"`
+	Default string            `yaml:"primary,omitempty"`
```

Comment:
This should also be default, not primary. We should only changing the name of the device to "default".

Reply:
It should be "primary" or something similar. It's neither the default network nor device, so using default is incorrect. The internal structs also should be changed, but I was leaving them intact or now because this involves a much more significant change and regular users won't see them.

Reply:
But that's my point, we should't be changing "default" to "primary" for this, as that is the name of the specific configuration that we have in the templates. Or we have to change it there too, but I think there is precedent there already not to change that.